internal/cli/atlas/metrics/databases/describe.go

```diff
@@ -59,14 +59,24 @@ func (opts *DatabasesDescribeOpts) Run() error {
 	return opts.Print(r)
 }
 
-// mcli om metric(s) disk(s) describe <hostId:port> <name> --granularity g --period p --start start --end end [--type type] [--projectId projectId].
-func DatabasesDescribeBuilder() *cobra.Command {
+// mcli atlas metric(s) database(s) describe <host:port> <databaseName> --granularity g --period p --start start --end end [--type type] [--projectId projectId].
+func DescribeBuilder() *cobra.Command {
 	const argsN = 2
-	opts := &DatabasesDescribeOpts{}
+	opts := &DescribeOpts{}
 	cmd := &cobra.Command{
-		Use:   "describe <hostId> <name>",
+		Use:   "describe <host:port> <databaseName>",
 		Short: "Describe database measurements for a given host database.",
 		Args:  require.ExactArgs(argsN),
+		Annotations: map[string]string{
+			"arg1":              "hostname:port",
+			"requiredArg1":      "hostname:port",
+			"hostname:portDesc": "Hostname and port number of the machine running the Atlas MongoDB process.",
+			"arg2":              "databaseName",
+			"requiredArg2":      "databaseName",
+			"databaseNameDesc":  "Label that identifies the database from which you want to retrieve measurements.",
+		},
```

Comment:
I see that the doc tool doesn’t recognize these args so there is something odd about this way of representing them.

Reply:
@gssbzn tagging you as you may know the correct way to annotate these commands

Reply:
sorry for the ping, I think I found out the correct way to annotate them